superset-frontend/.eslintrc.js

```diff
@@ -158,6 +158,10 @@ module.exports = {
           'error',
           { devDependencies: true },
         ],
+        '@typescript-eslint/no-unused-vars': [
+          'warn',
+          { argsIgnorePattern: '^_' },
+        ],
```

Comment:
..this might be unnecessary (see my other comment). However, I think this is a good ignore rule to have, but there may be others who have differing views.

Reply:
@villebro Reverted.